src/dfx/assets/new_project_motoko_files/README.md.patch

```diff
@@ -0,0 +1,7 @@
+--- a/README.md
++++ b/README.md
+@@ -9,2 +9,4 @@
+ - [Quick Start](https://internetcomputer.org/docs/quickstart/quickstart-intro)
+ - [SDK Developer Tools](https://internetcomputer.org/docs/developers-guide/sdk-guide)
++- [Motoko Programming Language Guide](https://internetcomputer.org/docs/current/developer-docs/build/cdks/motoko-dfinity/motoko/)
++- [Motoko Language Quick Reference](https://internetcomputer.org/docs/current/references/motoko-ref/)
```

Comment:
This link (https://internetcomputer.org/docs/current/references/motoko-ref/) currently redirects to https://internetcomputer.org/docs/current/motoko/main/base/ ("Motoko base library reference").  This is a list of base library types (a useful link to have in the readme, directly).

I imagine this is copied from somewhere.

The Motoko language quick reference url is: https://internetcomputer.org/docs/current/motoko/main/language-manual.

Reply:
It's what was in the original, I didn't check the file. They're also all old redirects, I'll update those too just in case they break someday.